config/default.yml

```diff
@@ -1471,6 +1471,11 @@ Lint/NextWithoutAccumulator:
   Enabled: true
   VersionAdded: '0.36'
 
+Lint/NonDeterministicRequireOrder:
+  Description: 'Always sort arrays returned by Dir.glob when requiring files.'
+  Enabled: true
+  VersionAdded: '0.78'
```

Comment:
I think this cop should be marked unsafe because it is not compatible behaviour.

```yaml
 VersionAdded: '0.78'
+Safe: false
```